docs/src/App.vue

```diff
@@ -33,6 +34,7 @@
 </template>
 
 <script>
+import tokenPackage from '../../package.json';
 import Tokens from './assets/cdr-tokens.json';
```

Comment:
Is having Tokens and tokenPackage temporary? Are we moving towards one over the other in the near future?

Reply:
I just added that to show whatever version is reflected on the gh-pages once we actually publish